featurebyte/feast/patch.py

```diff
@@ -25,10 +25,13 @@ def augment_response_with_on_demand_transforms(
     full_feature_names: bool,
 ) -> None:
     """
-    The main difference between this and the original Feast implementation is that this function explicitly
-    casts the transformed feature values to the expected data type. This is necessary because the
-    original implementation attempts to infer the data type from the values, which can lead to incorrect
-    data types being used.
+    The main difference between this and the original Feast implementation is that
+    * this function explicitly casts the transformed feature values to the expected data type. This is
+      necessary because the original implementation attempts to infer the data type from the values, which
+      can lead to incorrect data types being used.
+    * this function passes `include_event_timestamps=True` to the `to_dict` & `to_arrow` method of the
+      `OnlineResponse` object. This is necessary as the FeatureByte UDF needs to access the event timestamps
+      to correctly transform the features.
```

Comment:
For newer Feast, we also need to update `to_arrow` method so that it include event timestamp columns in the input dataframe.